PKCS12.xs

```diff
@@ -992,6 +996,18 @@ new(class)
   OUTPUT:
   RETVAL
 
+IV legacy_support(class)
```

Comment:
Again, maybe I should just update the object with this - but then again it looks like the Crypt::OpenSSL::PKCS12 is really just the openssl object and does not have any other attributes so this is likely bes